crates/config/src/config.rs

```diff
@@ -10,6 +10,12 @@ use std::{
     path::PathBuf,
 };
 
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Serialize, Deserialize, Hash)]
```

Comment:
Do we really need `Copy` here?

Reply:
```Copy``` for this simply enum is cheap, just like u8.